### ISSUE TITLE
When H800 is trained with FP8, the performance is not significantly improved compared to FP16, and is even worse than FP16.

### ISSUE THREAD

**Issue by yangzhipeng1108** [NVIDIA/Megatron-LM#860]
**Your question**
Ask a clear and concise question about Megatron-LM.

**Comment by Chaobai-Jiang** [comment]
hello，Please tell me, do you know how to start fp8 training? thanks

**Comment by yangzhipeng1108** [comment]
--bf16 \
        --fp8-format hybrid \
        --fp8-amax-compute-algo max \
        --fp8-amax-history-len 1024 \
        --transformer-impl transformer_engine

**Comment by Chaobai-Jiang** [comment]
> ```
>     --bf16 \
>     --fp8-format hybrid \
>     --fp8-amax-compute-algo max \
>     --fp8-amax-history-len 1024 \
>     --transformer-impl transformer_engine
> ```

thanks，I occur the same performance degradation problem as you on H20 GPU, please check your transformer engine version, upgrading to the new version may solve your problem

**Comment by yanchenmochen** [comment]
@LiaoYuanF thanks the parameter setting about fp8, if I want to make experiments which firstly trains llama2 using fp8 (25%)， then trains using bf16 (75%) how to initiate the trainging process

**Comment by qijiaxing** [comment]
> [@LiaoYuanF](https://github.com/LiaoYuanF) thanks the parameter setting about fp8, if I want to make experiments which firstly trains llama2 using fp8 (25%)， then trains using bf16 (75%) how to initiate the trainging process

you need to save checkpoint, stop training, turn off fp8, then start again

**Comment by Chaobai-Jiang** [comment]
> [@LiaoYuanF](https://github.com/LiaoYuanF) thanks the parameter setting about fp8, if I want to make experiments which firstly trains llama2 using fp8 (25%)， then trains using bf16 (75%) how to initiate the trainging process

You can separate the epochs for different accuracy （fp 8 or bf 16）training, although I don't understand what your intention is

**Comment by yanchenmochen** [comment]
@yangzhipeng1108 I tried initiate my fp8 experiment using the parameters, But unfortunately, The error OOM occur. Whe --bf16 succeed, but adding the fp8 parameter oom error occurs.
```
vocab_parallel_logits = vocab_parallel_logits.float()
torch.OutOfMemoryError:
```
